onnxruntime/python/_pybind_state.py

```diff
@@ -5,9 +5,23 @@
 
 import os
 import platform
+import sys
 import warnings
 import onnxruntime.capi._ld_preload  # noqa: F401
 
+# Python 3.8 (and later) on Windows doesn't search system PATH when loading DLLs,
+# so CUDA location needs to be specified explicitly.
+if platform.system() == "Windows" and sys.version_info >= (3, 8):
+    CUDA_VERSION = "10.2"
+    CUDNN_VERSION = "8"
+    cuda_env_variable = "CUDA_PATH_V" + CUDA_VERSION.replace(".", "_")
+    if cuda_env_variable not in os.environ:
+        raise ImportError(f"CUDA Toolkit {CUDA_VERSION} not installed on the machine.")
+    cuda_bin_dir = os.path.join(os.environ[cuda_env_variable], "bin")
+    if not os.path.isfile(os.path.join(cuda_bin_dir, f"cudnn64_{CUDNN_VERSION}.dll")):
```

Comment:
Hmm... In our CI build machines, cudnn is installed in a different folder.  I can change it, but it takes some time.

Reply:
Thank you @snnn. I believe this makes sense, as it is in line with official installation instructions from NVIDIA. It will be needed in order to produce an official package for Python 3.8 (https://github.com/microsoft/onnxruntime/issues/5963).